backend/test.sh

```diff
@@ -5,6 +5,8 @@ set -euo pipefail
 pip -q install --upgrade pip
 pip -q install --cache-dir=.pip -r requirements.txt
 
+pip -q install --cache-dir=.pip /lib/Akvo_rabbitmq_client
```

Comment:
`requirements.txt` ?

Reply:
Hi @kjkoster,

We tried to include the local packages in the requirements.txt file before, but we encountered some problems with the CI build. So, we are taking this approach to resolve it.

Thank you.